tests/integration/testlib/bondlib.py

```diff
@@ -64,5 +48,4 @@ def bond_interface(name, port, extra_iface_state=None, create=True):
                     }
                 ]
             },
-            verify_change=False,
```

Comment:
By omitting the verify_change, after applying, nmstate will verify whether the outcome network configuration matches with desired, if not rollback will happen. Out of curiosity, what is the benefit of setting `verify_change=False` previously?